content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -0,0 +1,66 @@
+---
+reviewers:
+- maplain
+title: Service Internal Traffic Policy
+content_type: concept
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Service Internal Traffic Policy_ enables cluster internal traffic routing to node
+local endpoints only. This approach provides a mechanism to help keep cluster
```

Comment:
Non-blocking nit, I found the first sentence confusing as to the benefit of ITP (I think mainly with local endpoint**s** being plural). I don't love my suggestion as it conflicts with the following sentence, but something similar to:

```suggestion
_Service Internal Traffic Policy_ enables restricted Service internal traffic to only the node it originated from. This approach provides a mechanism to help keep cluster
```

Blending the first two sentences together also might work!

Reply:
updated. let me know your thoughts :) 
@jimangel

Reply:
Reads a lot better IMO thanks!